RZTransitions-Demo/RZTransitions-Demo/ViewControllers/RZSimpleCollectionViewController.swift

```diff
@@ -0,0 +1,161 @@
+    , RZRectZoomAnimationDelegate
+{
+    @IBOutlet weak var collectionView: UICollectionView?
+
+    var circleTransitionStartPoint: CGPoint = CGPointZero
+    var transitionCellRect: CGRect = CGRectZero
+    var presentOverscrollInteractor: RZOverscrollInteractionController?
+    var presentDismissAnimationController: RZRectZoomAnimationController?
+
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: NSBundle?) {
+        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    override func viewDidLoad()
+    {
+        super.viewDidLoad()
+
+        self.collectionView?.registerClass(UICollectionViewCell.self, forCellWithReuseIdentifier:kRZCollectionViewCellReuseId);
+
+//     TODO: Currently the RZOverscrollInteractor will take over the collection view's delegate, meaning that ```didSelectItemAtIndexPath:```
+//     will not be forwarded back.  RZOverscrollInteractor requires a bit of a rewrite to use KVO instead of delegation to address this.
+//        self.presentOverscrollInteractor = [[RZOverscrollInteractionController alloc] init];
+//        [self.presentOverscrollInteractor attachViewController:self withAction:RZTransitionAction_Present];
+//        [self.presentOverscrollInteractor setNextViewControllerDelegate:self];
+//        [[RZTransitionsManager shared] setInteractionController:self.presentOverscrollInteractor
+//                                             fromViewController:[self class]
+//                                               toViewController:nil
+//                                                      forAction:RZTransitionAction_Present];
+
+    self.presentDismissAnimationController = RZRectZoomAnimationController()
+    self.presentDismissAnimationController?.rectZoomDelegate = self
+
+    RZTransitionsManager.shared().setAnimationController( self.presentDismissAnimationController,
+        fromViewController:self.dynamicType,
+        forAction:.PresentDismiss )
+
+    self.transitioningDelegate = RZTransitionsManager.shared()
+    }
+
+//    - (void)viewDidAppear:(BOOL)animated
+//{
+//    [super viewDidAppear:animated];
+//    // TODO: ** Cannot set the scroll view delegate and the collection view delegate at the same time **
+//    //    [self.presentOverscrollInteractor watchScrollView:self.collectionView];
+//}
+
+//MARK: - New VC Helper Methods
+
+    func newColorVCWithColor(color: UIColor?) -> UIViewController
+    {
+        let newColorVC = RZSimpleColorViewController(color: color)
+        newColorVC.transitioningDelegate = RZTransitionsManager.shared()
+
+        // TODO: Hook up next VC's dismiss transition
+        return newColorVC;
+    }
+
+//MARK: - UICollectionViewDelegate
+
+    func collectionView(collectionView: UICollectionView, didSelectItemAtIndexPath indexPath:NSIndexPath)
+    {
+        let cell = collectionView.cellForItemAtIndexPath(indexPath)!
```

Comment:
guard let this and throw a `fatalError()` with a valid message explaining why this happened (which also will unwrap cell and avoid force unwrapping cellBackgroundColor later